functions/delete_wordle.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+const gitOpsApi = require("git-ops-api");
+const {
+    Octokit
+} = require("@octokit/core");
+
+exports.handler = async function (event) {
+    console.log("Received delete_wordle request at " + new Date() + " with path " + event.path);
+    var branch = /delete_wordle\/(.*)/.exec(event.path)[1];
+
+    // Reject the request when:
+    // 1. Not a DELETE request;
+    // 2. Doesn’t provide required values
+    if (event.httpMethod !== "DELETE" || !branch ||
```

Comment:
Consolidate this function with the error checking at the head of create_branch.js